pkg/ansiescape/split.go

```diff
@@ -0,0 +1,89 @@
+package ansiescape
+
+import "bytes"
+
+// dropCR drops a terminal \r from the data.
+func dropCR(data []byte) []byte {
```

Comment:
I could see how this could be wrong in this case if there is both a leading and trailing `\r`, the original implementation only allowed to drop a single trailing value.